packages/xmppc/build.sh

```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=https://codeberg.org/Anoxinon_e.V./xmppc
+TERMUX_PKG_DESCRIPTION="Command Line Interface Tool for XMPP"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="j.r, @jugendhacker, j.r@jugendhacker.de"
```

Comment:
This field look a bit weird due to the commas. Most other maintainers are written as `name lastname @username` or `name lastname <name@email.com>`, please use one of those formats (for example `jugendhacker <j.r@jugendhacker.de>`).

Reply:
Changed it to `j.r <j.r@jugendhacker.de>`